### ISSUE TITLE
Publish as torcx module

### ISSUE THREAD

**Issue by paour** [zoobab/screen-static-coreos#5]
Could you publish screen as a torcx module, so it can be automatically installed by Ignition when Flatcar is run?

Would you accept a pull request if you'd rather not do it yourself?

**Comment by zoobab** [comment]
Go for a pull request, never used Ignition and torcx, will look at it.

**Comment by paour** [comment]
Here it is. If you merge it, you may need to enable GitHub Actions, and then tag the repo with `vA.B.C` the screen version number, and a release will automatically be built, as it was on my fork.

To install a Torcx module, see my other repo's README. https://github.com/paour/torcx-mosh-server

**Comment by zoobab** [comment]
I said to myself that I would never use Github Actions, because it creates a dependency towards proprietary services.

But I merged the request anyway, if it can be at least useful to you.

For other projects, I used a Dockerfile I could build at least on my laptop, and use a shell script to extract the binary:

https://github.com/zoobab/DirtyJTAG/blob/master/Dockerfile
https://github.com/zoobab/DirtyJTAG/blob/master/build.sh

**Comment by paour** [comment]
I agree that GH actions are yet another dependency that I'd rather avoid (and I use Bitrise and GitLab at work :-) ). However, they seem to be pretty well thought out, and you can run them locally https://github.com/nektos/act too.

I like the Docker trick, but wouldn't it be more "dockery" to run make as part of the container's execution rather than its build?
